internal/transport/grpchttp2/http2bridge.go

```diff
@@ -0,0 +1,214 @@
+/*
+ *
+ * Copyright 2024 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package grpchttp2
+
+import (
+	"io"
+
+	"golang.org/x/net/http2"
+	"golang.org/x/net/http2/hpack"
+	"google.golang.org/grpc"
+)
+
+const (
+	initHeaderTableSize = 4096
+	maxFrameLen         = 16384
+)
+
+type HTTP2FramerBridge struct {
```

Comment:
s/HTTP2FramerBridge/FramerBridge

See: go/go-style/decisions#repetition

TL;dr
The current type name would be used as `grpchttp2.HTTP2FramerBridge`, while my suggestion would be used as `grpchttp2.FramerBridge`

Reply:
The `HTTP2` part mainly referenced the `http2` package, but I had to leave it all in caps due to the type being exported. Do you think `FramerBridge` is a better name?

Reply:
Yes, I feel `FramerBridge` is good, since an exported symbol is always used with the package name prefixed to it.